modules/msk/genomenexus/vcf2maf/main.nf

```diff
@@ -0,0 +1,45 @@
+process GENOMENEXUS_VCF2MAF {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/genomenexus:latest':
+        'ghcr.io/msk-access/genomenexus:latest' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("vcf2maf_output/${meta.id}.maf"), emit: maf
```

Comment:
do we want to return all content from this folder?

Reply:
@buehlere  there's only one output MAF created in this folder, but the code loaded in the genome nexus docker image doesn't print an output file on it's own, and defaults to printing the output file in the vcf2maf_output folder. happy to discuss this more on a call. I've noticed this default output directory is the code from the genome nexus team and I didn't want to mess with that